CBLClient/Apps/CBLTestServer-Android/app/build.gradle

```diff
@@ -59,4 +82,7 @@ dependencies {
     implementation 'org.nanohttpd:nanohttpd:2.3.2-SNAPSHOT'
 
     implementation "com.couchbase.lite:couchbase-lite-android-ee:${COUCHBASE_LITE_VERSION}"
+
```

Comment:
Is this necessary?  You already have the source included above...

Reply:
thanks, I removed it